pkg/image/apiserver/admission/imagepolicy/imagepolicy.go

```diff
@@ -236,11 +179,6 @@ func (a *imagePolicyPlugin) admit(attr admission.Attributes, mutationAllowed boo
 		return nil
 	}
 
-	// This will convert any non-legacy Origin resource to a legacy resource, so specifying
-	// a 'builds.build.openshift.io' is converted to 'builds'.
-	// TODO: denormalize this at config time, or write a migration for user's config
-	attr = mutateAttributesToLegacyResources(attr)
```

Comment:
sounds like *this* was the culprit?

Reply:
yeah, probably answers the "why attrs comes without group" ;-)

Reply:
I agree